src/mono/wasm/build/WasmApp.targets

```diff
@@ -72,13 +71,23 @@
     <_ExeExt Condition="$([MSBuild]::IsOSPlatform('WINDOWS'))">.exe</_ExeExt>
   </PropertyGroup>
 
+  <PropertyGroup Condition="'$(EmscriptenSdkToolsPath)' == '' and '$(EMSDK_PATH)' != ''">
+    <EmscriptenSdkToolsPath>$([MSBuild]::EnsureTrailingSlash($(EMSDK_PATH)))</EmscriptenSdkToolsPath>
+    <EmSdkUpstreamBinPath>$([MSBuild]::NormalizeDirectory($(EmscriptenSdkToolsPath), 'upstream', 'bin'))</EmSdkUpstreamBinPath>
+
+    <EmscriptenNodeToolsPath>$([MSBuild]::NormalizeDirectory($(EmscriptenSdkToolsPath), 'node', '12.18.1_64bit'))</EmscriptenNodeToolsPath>
+    <EmscriptenPythonToolsPath>$([MSBuild]::NormalizeDirectory($(EmscriptenSdkToolsPath), 'python', '3.7.4-2_64bit'))</EmscriptenPythonToolsPath>
```

Comment:
the versions should probably explicit props and a check to make sure it exists

Reply:
This is closely tied to Emscripten 2.0.12, so the paths won't change separately I think. I'll add the Exists checks.

Reply:
That was the point I was making, they version with the emsdk version so let's make sure the build breaks in a way that is obvious

Reply:
And having the versions there at all caused an issue:
- CI has Emscripten 2.0.14
- The packages that I was testing with was 2.0.12

.. which had differing python/node versions, .. and that caused the build to fail!